src/Explorer/Menus/CommandBar/CommandBarComponentButtonFactory.test.ts

```diff
@@ -368,62 +366,6 @@ describe("CommandBarComponentButtonFactory tests", () => {
     });
   });
 
-  describe("GitHub buttons", () => {
```

Comment:
Why are we commenting out the other changes but fully removing this one?

Reply:
I got rid of the tests but in the code - I will ultimately check to see if there are any dependencies/references to the commented portion so I can remove them easily all together. So, it'd be better for me to track it this way.